cmake/Dependencies.cmake

```diff
@@ -459,9 +459,7 @@ if(BUILD_CAFFE2 OR BUILD_ATEN)
     set(HIP_HIPCC_FLAGS "${HIP_HIPCC_FLAGS} -Wno-unused-command-line-argument")
 
     set(Caffe2_HIP_INCLUDES
-      ${hip_INCLUDE_DIRS} ${rocrand_INCLUDE_DIRS} ${hiprand_INCLUDE_DIRS} ${rocblas_INCLUDE_DIRS} ${miopen_INCLUDE_DIRS} ${thrust_INCLUDE_DIRS} $<INSTALL_INTERFACE:include> ${Caffe2_HIP_INCLUDES})
-    # This is needed for library added by hip_add_library (same for hip_add_executable)
-    hip_include_directories(${Caffe2_HIP_INCLUDES})
```

Comment:
this is better to leave here as it applies to all libraries/executable added via hip_add_library/hip_add_executable. 
Caffe2 has two hip_add_library (libcaffe2_hip.so and the python binding) and several hip_add_executable (c++ tests)

Reply:
I see. In that case let's just move it from Dependencies to caffe2/CMake as what you have done initially (add a note to prevent other people to move it back again :-))